views/partials/dashboards/breaches-dash.hbs

```diff
@@ -1,8 +1,12 @@
 
-<main id="dashboard" class="dashboard bg-white clear-header">
+<main id="dashboard" class="dashboard clear-header">
   <div class="row jst-cntr">
     <div class="col-8 no-vertical-padding">
-      <span class="welcome-back">{{{fluentNestedBold "welcome-back" userName=req.session.user.fxa_profile_json.email }}}</span>
+      {{#if req.session.newUser}}
+        <span class="welcome-user">{{{fluentNestedBold "welcome-back" userName=req.session.user.fxa_profile_json.email }}}</span>
```

Comment:
Should this `welcome-user` span use `{{{fluentNestedBold "welcome-user" ...` (instead of the same `welcome-back` as below?)

Reply:
yes!